src/main/java/eu/deltasource/internship/hotelmanagementsystem/Hotel.java

```diff
@@ -0,0 +1,97 @@
+package eu.deltasource.internship.hotelmanagementsystem;
+
+import eu.deltasource.internship.hotelmanagementsystem.hotel.service.domain.commodities.AbstractCommodity;
+import eu.deltasource.internship.hotelmanagementsystem.hotel.service.domain.commodities.Bed;
+import eu.deltasource.internship.hotelmanagementsystem.hotel.service.domain.commodities.Booking;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Represents a hotel with rooms
+ */
+public class Hotel {
+
+    private String name;
+    private List<Room> rooms;
+
+    /**
+     * Parameterized constructor
+     */
+    public Hotel(String name, List<Room> rooms) {
```

Comment:
same remark here, let's initialize the collections when we create the objects, inside the constructors.f.e.:
`public Hotel (String name) {
this.name = name;
this.rooms = new ArrayList();
}`